index.html

```diff
@@ -91,7 +91,7 @@
         featured ? "featured" : "",
       ].join(" ");
     %>
-    <div id="<%= id %>" class="<%= classNames %>" data-id="<%= id %>" data-timestamp="<%= rawTimestamp %>">
+    <div id="<%- id %>" class="<%- classNames %>" data-id="<%- id %>">
```

Comment:
yup, we never used that before. I think I added it just for testing and forgot to revert it back after.